doc/releases/migration-guide-4.1.rst

```diff
@@ -260,6 +260,20 @@ Other Subsystems
 Flash map
 =========
 
+Secure storage
+==============
+
+* The :kconfig:option:`CONFIG_SECURE_STORAGE_ITS_STORE_MODULE` symbol no longer implies the
+  :kconfig:option:`CONFIG_FLASH` Kconfig option. Platforms using Secure storage must explicitly enable
+  :kconfig:option:`CONFIG_FLASH` Kconfig option.
+
+* The :kconfig:option:`CONFIG_SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS` symbol no longer implies the
+  :kconfig:option:`CONFIG_FLASH_MAP` and :kconfig:option:`CONFIG_NVS` Kconfig options.
+  It either no longer selects :kconfig:option:`CONFIG_SETTINGS` Kconfig option.
+  Platforms using Secure storage must explicitly enable :kconfig:option:`CONFIG_FLASH_MAP`,
```

Comment:
Using Secure storage with the settings-based ITS store module.

Reply:
didn't get this. How the final text should look like?

Reply:
Actually I think it can be made as simple as the following as we previously refer to `CONFIG_SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS`:
```suggestion
  Platforms using it must now explicitly enable :kconfig:option:`CONFIG_FLASH_MAP`,
```